chrome/content/zotero/elements/annotationRow.js

```diff
@@ -96,7 +96,18 @@
 			if (this._annotation.annotationText) {
 				let text = document.createElement('div');
 				text.classList.add('quote');
-				text.innerHTML = this._annotation.annotationText;
+				// Try to sanitize annotation text
+				let parserUtils = Cc["@mozilla.org/parserutils;1"].getService(Ci.nsIParserUtils);
+				let sanitizedHtml = parserUtils.sanitize(this._annotation.annotationText,
+					Ci.nsIParserUtils.SanitizerDropForms | Ci.nsIParserUtils.SanitizerDropMedia);
+				// The content will be wrapped in <body></body>
+				let sanitizedBody = sanitizedHtml.match(/<body[^>]*>([\s\S]*?)<\/body>/i);
+				if (sanitizedBody) {
+					text.innerHTML = sanitizedBody[0];
+				}
+				else {
+					text.textContent = this._annotation.annotationText;
+				}
```

Comment:
Sorry, I'm a bit confused... After our discussion [last week](https://github.com/zotero/zotero/pull/3416#discussion_r1727941691), I pushed https://github.com/zotero/zotero/pull/3416/commits/4bc0393530a1bdf21712f20139906f9af5a36597 to just strip all html tags and render everything as plain text. That is, until all relevant PRs such as https://github.com/zotero/zotero/pull/4607 are merged to avoid conflicts. So all that should be happening now is a [call](https://github.com/zotero/zotero/blob/e28002bd2c7ef72a5df4b13de4ee8b0850c1a81f/chrome/content/zotero/elements/annotationRow.js#L103) to `convertToPlainText`.

This is what this part looks like this to me on this branch:
```
if (this._annotation.annotationText) {
  let text = document.createElement('div');
  text.classList.add('quote');
  let plainQuote = parserUtils.convertToPlainText(this._annotation.annotationText, Ci.nsIDocumentEncoder.OutputRaw, 0);
  text.textContent = plainQuote;
  this._body.append(text);
}
``` 

Is that not what is appearing on your end?

Reply:
Gotcha! No worries!